image-classifier/app.R

```diff
@@ -2,89 +2,110 @@ library(magrittr)
 library(reticulate)  # Used to call Tensorflow Python script
 library(shiny)
 library(shinycssloaders)
+library(gitlink)
```

Comment:
Since this package is not on CRAN, it's an extra step for users to have to figure out where it lives and install it, so I would rather not include it.

Reply:
That's a great call. I'll remove for now - plan is to put it on CRAN :wink: